peachjam/models/core_document_model.py

```diff
@@ -218,6 +218,7 @@ class CoreDocument(PolymorphicModel):
 
     created_at = models.DateTimeField(_("created at"), auto_now_add=True)
     updated_at = models.DateTimeField(_("updated at"), auto_now=True)
+    allow_robots = models.BooleanField(_("allow robots"), default=True)
```

Comment:
Two suggestions:

1. add `db_index=True` so that it's indexed by the db. We don't want to have to go through 100,000s of rows in the db just to find the 10s that have this set to False.
2. add `help_text=_("Allow this document to be indexed by search engine robots.")`